OCHuds/Classes/AnimatedBaseHUD.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  AnimatedBaseHUD.swift
+//  Pods
+//
+//  Created by Giovane Berny Possebon on 16/08/16.
+//
+//
+
+import UIKit
+
+public class AnimatedBaseHUD {
```

Comment:
@Morbix I tried to do that way, but it can limit us... 'Cause if all HUD uses the same Base, them all have the same functions, but that's not the situation, or do you have a solution for that? For example, I don't want to be possible to something like "SpinnerHUD.show(withImages: images)" you know?